src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.mainContainer {
+  display: flex;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
```

Comment:
why do you need flex here?

Reply:
I added selector for header__logo class. I have to use flex in `.header__logo` because it hapled me to align logo in center.